solr/solrj/src/test/org/apache/solr/client/solrj/io/graph/GraphExpressionTest.java

```diff
@@ -1075,15 +1075,15 @@ public void testGatherNodesFriendsStream() throws Exception {
     tuples = getTuples(stream);
 
     tuples.sort(new FieldComparator("node", ComparatorOrder.ASCENDING));
-    assertTrue(tuples.size() == 4);
-    assertTrue(tuples.get(0).getString("node").equals("bill"));
-    assertTrue(tuples.get(0).getLong("level").equals(0L));
-    assertTrue(tuples.get(1).getString("node").equals("jim"));
-    assertTrue(tuples.get(1).getLong("level").equals(1L));
-    assertTrue(tuples.get(2).getString("node").equals("max"));
-    assertTrue(tuples.get(2).getLong("level").equals(1L));
-    assertTrue(tuples.get(3).getString("node").equals("sam"));
-    assertTrue(tuples.get(3).getLong("level").equals(1L));
+    assertEquals(4, tuples.size());
+    assertEquals("bill", tuples.get(0).getString("node"));
+    assertEquals(0L, (long) tuples.get(0).getLong("level"));
```

Comment:
I think so - I'll take a look

Reply:
Hmmm so `tuple.getLong` returns a `Long` because it also returns `null` sometimes. Changing to `long` can't be done for `tuple.getLong` since then `null` can't be returned. `3L` is still a `long` and can't be compared with `Long` apparently. I could change it to

```
assertEquals(Long.valueOf(3), tuple.getLong("myId"))
```

but that seems worse.

I don't have any other ideas right now :/